keanu-python/tests/test_vertex.py

```diff
@@ -70,7 +70,53 @@ def test_vertex_can_observe_ndarray(jvm_view):
     ndarray = np.array([[1.,2.]])
     gaussian.observe(ndarray)
 
-    nd4j_tensor_flat = gaussian.getValue().asFlatArray()
+    nd4j_tensor_flat = gaussian.get_value().asFlatArray()
     assert nd4j_tensor_flat[0] == 1.
     assert nd4j_tensor_flat[1] == 2.
-    
\ No newline at end of file
+
+
+def test_get_connected_graph(jvm_view):
+    gaussian = kn.Vertex(jvm_view.GaussianVertex, (0., 1.))
+    connected_graph = gaussian.get_connected_graph()
+
+    assert len(connected_graph) == 3
+
+
+def test_id_str_of_downstream_vertex_is_higher_than_upstream(jvm_view):
+    hyper_params = kn.Vertex(jvm_view.GaussianVertex, (0., 1.))
+    gaussian = kn.Vertex(jvm_view.GaussianVertex, (0., hyper_params))
+
+    hyper_params_id = hyper_params.get_id()
+    gaussian_id = gaussian.get_id()
+
+    assert type(hyper_params_id) == str
+    assert type(gaussian_id) == str
+
+    assert hyper_params_id < gaussian_id
+
+
+def test_construct_vertex_with_java_vertex(jvm_view):
+    java_vertex = kn.Vertex(jvm_view.GaussianVertex, (0., 1.)).unwrap()
+    python_vertex = kn.Vertex(java_vertex=java_vertex)
+
+    assert java_vertex.getId().toString() == python_vertex.get_id()
+
+
+def test_java_list_to_python_list(jvm_view):
+    gaussian = kn.Vertex(jvm_view.GaussianVertex, (0., 1.))
```

Comment:
can you do pass `gaussian.get_connected_graph()` into the test for the `list` and `set`? They're collections so good to test with > 1 element

Reply:
`getConnectedGraph` returns a set, so the two tests are testing:

java set -> python set
java list -> python list

But good point, I'll pass a list with more than one element.